tests/pdfExtractor.test.js

```diff
@@ -0,0 +1,45 @@
+const { extractTextFromPdf } = require("../backend/pdfExtractor");
+const fs = require('fs');
+const path = require('path');
+
+const loadPdf = (fileName) => {
+  const filePath = path.join(__dirname, "pdfFiles", fileName);
+  return fs.readFileSync(filePath);
+};
```

Comment:
As we use this approach multiple times, I created a function so we can save some space.